static/app/views/issueList/customViewsHeader.tsx

```diff
@@ -0,0 +1,301 @@
+import {useEffect, useState} from 'react';
+import type {InjectedRouter} from 'react-router';
+import styled from '@emotion/styled';
+import {debounce} from 'lodash';
+
+import GlobalEventProcessingAlert from 'sentry/components/globalEventProcessingAlert';
+import * as Layout from 'sentry/components/layouts/thirds';
+import {PageHeadingQuestionTooltip} from 'sentry/components/pageHeadingQuestionTooltip';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types/organization';
+import {useNavigate} from 'sentry/utils/useNavigate';
+import useProjects from 'sentry/utils/useProjects';
+import {
+  DraggableTabBar,
+  type Tab,
+} from 'sentry/views/issueList/groupSearchViewTabs/draggableTabBar';
+import {useUpdateGroupSearchViews} from 'sentry/views/issueList/mutations/useUpdateGroupSearchViews';
+import {useFetchGroupSearchViews} from 'sentry/views/issueList/queries/useFetchGroupSearchViews';
+import type {GroupSearchView} from 'sentry/views/issueList/types';
+
+import {IssueSortOptions, type QueryCounts} from './utils';
+
+type CustomViewsIssueListHeaderProps = {
+  organization: Organization;
+  queryCounts: QueryCounts;
+  router: InjectedRouter;
+  selectedProjectIds: number[];
+  initalView?: GroupSearchView;
+};
+
+type CustomViewsIssueListHeaderTabsContentProps = {
+  organization: Organization;
+  queryCounts: QueryCounts;
+  router: InjectedRouter;
+  views: GroupSearchView[];
+};
+
+function CustomViewsIssueListHeader({...props}: CustomViewsIssueListHeaderProps) {
```

Comment:
i'd take out the props that the sub component isn't asking for
```suggestion
function CustomViewsIssueListHeader({selectedProjectIds, ...props}: CustomViewsIssueListHeaderProps) {
```